spec.bs

```diff
@@ -1250,6 +1285,7 @@ On the other hand, methods for getting data from the [=shared storage database=]
     boolean resolveToConfig = false;
     boolean keepAlive = false;
     SharedStoragePrivateAggregationConfig privateAggregationConfig;
+    DOMString savedQuery = "";
```

Comment:
I think usually we omit the `= ""` for string default values.

Reply:
Done.